Std/Data/Fin/Basic.lean

```diff
@@ -9,6 +9,12 @@ namespace Fin
 /-- `min n m` as an element of `Fin (m + 1)` -/
 def clamp (n m : Nat) : Fin (m + 1) := ⟨min n m, Nat.lt_succ_of_le (Nat.min_le_right ..)⟩
 
+/-- `enum n` is the array of all elements of `Fin n` in order -/
+def enum (n) : Array (Fin n) := Array.ofFn id
+
+/-- `list n` is the list of all elements of `Fin n` in order -/
+def list (n) : List (Fin n) := (enum n).data
```

Comment:
This function already exists in Mathlib as `List.finRange`.  Was the new choice of name deliberate?

Reply:
No, I'll fix that.

Reply:
See #899